### ISSUE TITLE
Place subtasks in general task manager view

### ISSUE THREAD

**Issue by eshwarnadh** [eshwarnadh/flairtech-vs#99]
Place all the subtasks which are under a parent task in Task manager _i.e_  list both tasks and subtasks in same ui

**Comment by tejeswar2020** [comment]
Yes all tasks and subtasks should be shown in same ui (Dashboard itself)
It is also better to show the subtasks under Parent task.
